src/editor/Collaboration.js

```diff
@@ -125,6 +134,29 @@ define(function (require, exports, module) {
             }
         }
         _changing = false;
+        peer.on('fileTransfer', function (metadata, receiver) {
```

Comment:
nit: try to stick to one kind of quoting system in your file. Either use double or single everywhere in this file. Since you seem to be using double in previous commits, stay consistent with that.